templates/default/twofactorauth/twofactorauthcode.html

```diff
@@ -78,7 +78,7 @@
 						{trans("Authentication code:")}
 					</td>
 					<td>
-						<input class="bold" type="number" name="authcodeform[authcode]" size="10" accesskey="l" autofocus autocomplete="off">
+						<input class="bold" type="number" name="authcodeform[authcode]" size="10" accesskey="l" autofocus autocomplete="false">
 					</td>
```

Comment:
W dokumentacji:
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete
nie znalazłem obsługi wartości `false`...

Reply:
To jest taki hack na chrome, który znalazłem - https://stackoverflow.com/questions/12374442/chrome-ignores-autocomplete-off

Inne opcje działające:
- ustawienie type="search" i autocomplete="off",
- dodanie atrybutu readonly i usuwanie go przy onfocus,

Reply:
Ten PR rozwiązuje problem autow
Której z podanych metod mam użyć?

Reply:
Na pewno `autocomplete="off"` nie działa w Chrome 91?

Reply:
Gdyby mnie to nie wkurzało to bym tego nie tykał... :)
 Mam obecnie 91.0.4472.77

Reply:
Jak zachowuje się `autocomplete="off"` w FF?

Reply:
1. https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion#the_autocomplete_attribute_and_login_fields
2. https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete#values (i niebieska ramka przy opisie `off`.

Reply:
W FF działa autocomplete="off" tak samo jak i autocomplete="false".

Reply:
Przeczytaj informacje pod linkami, które podałem. Nie możemy ot tak po prostu użyć nielegalnej wartości `false`.
Z linków powyżej wynika, że chyba powinniśmy użyć `new-password` zamiast `false`.

Reply:
https://github.com/lmsgit/lms/pull/2039/commits/5d4c64693b46f40bd66cd319c7430984ee780403